packages/esm-bed-management-app/src/bed-administration/bed-administration-table.component.tsx

```
@@ -255,6 +248,7 @@ const BedAdministrationTable: React.FC = () => {
           )}
         </DataTable>
       </div>
+      <WorkspaceContainer contextKey="bed-administration" />
```

The `WorkspaceContainer` component should be moved to the root level (`root.component.tsx`) and use the correct `contextKey` to enable workspace functionality across all routes. Additionally, the `contextKey` must match a segment in the URL path. Using `bed-administration` would only work on the `/bed-administration` route and cause silent failures on other routes like `/location/:location`. Using `bed-management` (which matches the app's basename) ensures the workspace works on all routes.

```tsx
// root.component.tsx
<BrowserRouter basename={bedManagementBasename}>
  <LeftPanel />
  <main className={styles.container}>
    <Routes>
      <Route path="/" element={<Home />} />
      <Route path="/location/:location" element={<WardWithBeds />} />
      <Route path="/bed-administration" element={<BedAdministrationTable />} />
      <Route path="/bed-tags" element={<BedTagAdministrationTable />} />
      <Route path="/bed-types" element={<BedTypeAdministrationTable />} />
    </Routes>
  </main>
  <WorkspaceContainer contextKey="bed-management" />
</BrowserRouter>
```